### ISSUE TITLE
add CSV and Excel importer for customers

### ISSUE THREAD

**Issue by enesaktay** [FriendsOfSylius/SyliusImportExportPlugin#133]

**Comment by stevangoldduck** [comment]
I think we can add it by ourself following the Development documentation ??

**Comment by stevangoldduck** [comment]
![image](https://user-images.githubusercontent.com/36699633/48546790-ca1af600-e8fb-11e8-9174-d60446da44ca.png)
Available importers show that in my app may upload with Excel, but always show Skip 1.

**Comment by solutionDrive-Alt** [comment]
@stevangoldduck Just guessing after having looked at the code, but maybe the empty values in `Birthday` could be a problem. 
The skipped value is only set (in the ResourceImporter)[https://github.com/FriendsOfSylius/SyliusImportExportPlugin/blob/master/src/Importer/ResourceImporter.php#L112], if ha row is `incomplete`.
My next guess would be that the `FALSE` in `Subscribed_to_newsletter` could be a problem.

Have you tried it with a csv-file yet, maybe it is a xlxs-related problem.

**Comment by stevangoldduck** [comment]
I just wanna close this.
1. Follow the Development guide for Importer
2. Just make sure your file as same as like this picture below for Excel
![image](https://user-images.githubusercontent.com/36699633/48615539-c2bf1f80-e9c3-11e8-91c7-073aaf228567.png)
Match the column with your _HeaderKey in Generic ResourceProcessor_ 
3. For CSV file format, you can check the examples in the _tests/Behat/Resources/fixtures_

**Comment by stevangoldduck** [comment]
@enesaktay

**Comment by enesaktay** [comment]
Did you try and remove the first row and move everything else up so the table starts at 1-A?

**Comment by stevangoldduck** [comment]
Yeah i've tried it and still skipped. With the example above, the data can be imported.
